src/main/flight/servos.c

```diff
@@ -296,20 +296,54 @@ STATIC_UNIT_TESTED void forwardAuxChannelsToServos(uint8_t firstServoIndex)
     }
 }
 
+static void biquadFilterInitd1d2(biquadFilter_t *filter, float sample)
+{
+    filter->d2 = filter->b2 * sample - filter->a2 * sample;
+    filter->d1 = filter->b1 * sample - filter->a1 * sample + filter->d2;
+}
+
 static void filterServos(void)
 {
-    if (servoConfig()->servo_lowpass_freq) {
+    static uint16_t rudderFilterFreq;
+
+    if (servoConfig()->servo_lowpass_freq || servoConfig()->tri_servo_lpf_hz) {
         // Initialize servo lowpass filter (servos are calculated at looptime rate)
         if (!servoFilterIsSet) {
             for (int i = 0; i < MAX_SUPPORTED_SERVOS; i++) {
-                biquadFilterInitLPF(&servoFitlerState[i], servoConfig()->servo_lowpass_freq, gyro.targetLooptime);
+                biquadFilterInitLPF(&servoFilterState[i], servoConfig()->servo_lowpass_freq, gyro.targetLooptime);
+                biquadFilterInitd1d2(&servoFilterState[i], (float)servo[i]);
             }
+            rudderFilterFreq = servoConfig()->servo_lowpass_freq;
             servoFilterIsSet = true;
         }
 
+        bool rudderFilterEnable = false;
+        if (servoConfig()->tri_servo_lpf_hz) {
+            if (mixerConfig()->mixerMode == MIXER_TRI || mixerConfig()->mixerMode == MIXER_CUSTOM_TRI) {
+                int8_t throttlePercent = calculateThrottlePercent();
+                if (!ARMING_FLAG(ARMED) || (throttlePercent >=0 && throttlePercent < 20)) rudderFilterEnable = true;
```

Comment:
I prefer to avoid magical constants in the code. 20% throttle is 

1. Inconsistent with how all throttle values are set in INAV (they use raw throttle values in [1000;2000] range
1. Very platform-specific. What if we have a seriously overpowered tricopter that hovers at 20% throttle already?